tools/AzureMarkdownRewriterTool/Program.cs

```diff
@@ -347,15 +349,24 @@ private bool CheckParameters()
         }
 
         // Removes "<br/>" or "<br />" before "[AZURE.INCLUDE".
-        private string FixAzureIncludeSyntax(string source)
+        private string FixAzureIncludeWithPrefixSyntax(string source)
+        {
+            return _azureHtmlIncludeWithPrefixRegex.Replace(source, "$2");
+        }
+
+        // Moves the content after "[AZURE.INCLUDE" section to the next line.
+        // For example, "[AZURE.INCLUDE [active-directory-devquickstarts-switcher](../../includes/active-directory-devquickstarts-switcher.md)] test"
+        // should be converted to 
+        // "[AZURE.INCLUDE [active-directory-devquickstarts-switcher](../../includes/active-directory-devquickstarts-switcher.md)]\r\ntest"
+        private string FixAzureIncludeWithPostfixSyntax(string source)
```

Comment:
This change will make inline include rule to block include rule. If the include thing is a token, may bring some side effect.